scripts/release/version.js

```diff
@@ -80,7 +80,7 @@ function getPackagesToUpdate() {
 
     const workspacePkgs = rootPackageJson.workspaces.reduce(
         (accWorkspacePkgs, workspace) => {
-            const workspacePkg = glob.sync(`${workspace}/package.json`);
+            const workspacePkg = globSync(`${workspace}/package.json`);
```

Comment:
We are using `glob.sync` in [a few other places as well](https://github.com/search?q=repo%3Asalesforce%2Flwc%20glob.sync&type=code).